### PR TITLE
Chore: Remove gf-form from datasource/loki and datasource/cloud-monitoring

### PR DESCRIPTION
**Pull request by khushijain21**
Part of #65513 

image

### CODE REVIEW

File: public/app/plugins/datasource/cloud-monitoring/components/AnnotationsHelp.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 export const AnnotationsHelp = () => {
   return (
-    <div className="gf-form grafana-info-box alert-info">
```
Comment (matyax) [inline public/app/plugins/datasource/cloud-monitoring/components/AnnotationsHelp.tsx:5]: This is not from the Loki data source, so maybe either revert the change or update the PR title and description please. Thank you!

File: public/app/plugins/datasource/loki/components/LokiQueryField.tsx [hunk 2]
```diff
@@ -70,11 +71,8 @@ export class LokiQueryField extends React.PureComponent<LokiQueryFieldProps, Lok
 
     return (
       <>
-        <div
-          className="gf-form-inline gf-form-inline--xs-view-flex-column flex-grow-1"
-          data-testid={this.props['data-testid']}
-        >
-          <div className="gf-form--grow flex-shrink-1 min-width-15">
+        <InlineFieldRow data-testid={this.props['data-testid']}>
```
Comment (matyax) [inline public/app/plugins/datasource/loki/components/LokiQueryField.tsx:74]: This change is breaking the query editor when you resize it.

imagen

imagen

**Review by matyax — CHANGES_REQUESTED** [review]
These changes are breaking the query field when you resize it.

**Comment by matyax** [comment]
Thank you so much for the contribution @khushijain21 ! Changes to the Loki Query Field component have the potential of breaking the UI in multiple places, so any change requires a very thorough testing. We can continue iterating, or you have the option to skip changes in that file.

**Comment by matyax** [comment]
Thanks for following this up @khushijain21 ! We will be reviewing as soon as we have capacity.

File: public/app/plugins/datasource/loki/components/AnnotationsQueryEditor.tsx [hunk 3]
```diff
@@ -49,8 +49,8 @@ export const LokiAnnotationsQueryEditor = memo(function LokiAnnotationQueryEdito
     queryType: annotation.queryType,
   };
   return (
-    <>
-      <div className="gf-form-group">
```
Comment (matyax) [inline public/app/plugins/datasource/loki/components/AnnotationsQueryEditor.tsx:53]: imagen

File: public/app/plugins/datasource/loki/components/LokiOptionFields.tsx [hunk 4]
```diff
@@ -82,18 +81,9 @@ export function LokiOptionFields(props: LokiOptionFieldsProps) {
   }
 
   return (
-    <div aria-label="Loki extra field" className="gf-form-inline">
+    <Stack alignItems="flex-start" gap={0.5} aria-label="Loki extra field">
```
Comment (matyax) [inline public/app/plugins/datasource/loki/components/LokiOptionFields.tsx:84]: imagen

File: public/app/plugins/datasource/cloud-monitoring/components/AnnotationsHelp.tsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
...
+    <div className="grafana-info-box alert-info">
```
Comment (matyax) [inline public/app/plugins/datasource/cloud-monitoring/components/AnnotationsHelp.tsx:5]: imagen

**Review by matyax — APPROVED** [review]

**Comment by matyax** [comment]
Thanks for your contribution @khushijain21 . If I may suggest, whenever you make a UI change, please add a few screenshots (ideally before/after) to make reviewing easier, I would appreciate that a lot.

**Comment by khushijain21** [comment]
> Thanks for your contribution @khushijain21 . If I may suggest, whenever you make a UI change, please add a few screenshots (ideally before/after) to make reviewing easier, I would appreciate that a lot.

Hello @matyax. Yes right. Will keep that in mind next time

**Comment by matyax** [comment]
@khushijain21 Would you mind merging the latest main to this branch please?

**Pull request merged**